inc/html.class.php

```diff
@@ -3889,6 +3889,7 @@ static function initEditorSystem($name, $rand = '', $display = true, $readonly =
             skin_url: '".$CFG_GLPI['root_doc']."/css/tiny_mce/skins/light',
             content_css: '$darker_css,".$CFG_GLPI['root_doc']."/css/tiny_mce_custom.css',
             cache_suffix: '?v=".GLPI_VERSION."',
+            min_height: '148px',
```

Comment:
```suggestion
            min_height: '150px',
```

Reply:
I strictly picked the value I observed .. but lets round this :)